### PR TITLE
mdct 2249 - change deviations section to only include radio buttons and a textarea

### PR DESCRIPTION
**Pull request by britt-mo**
### Description
This is a change only to 2023 -- when asked if there are deviations from the measure specifications, the user is prompted to answer Yes or No. If 'yes', a textArea appears and they need to describe the deviation. If they do not fill out the textArea, an error will appear at the bottom after the user clicks the `validate` button.

### Related ticket(s)

MDCT-2249

---
### How to test
Go into a measure (AAB-AD will work). Fill out the page and after you get to 'Performance Measures' you'll see the ability to add rates. Fill out one of the rates and then after you'll see a section called 'Deviations from Measure Specifications' click 'yes', then you should only see at textArea. See that leaving it blank will result in a validation error after clicking 'validate', and see that no error occurs when you fill out the textArea. 

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/measures/2023/shared/CommonQuestions/types.ts [hunk 1]
```diff
@@ -182,13 +182,16 @@ export interface complexRateFields {
...
 
+export interface DeviationField {
+  [DC.DEVIATION_REASON]: string;
+}
 export interface DeviationFields {
   [DC.OPTIONS]: string[];
   [DC.DENOMINATOR]: string;
   [DC.NUMERATOR]: string;
   [DC.OTHER]: string;
+  [DC.REASON]: string;
```
Comment (ntsummers1) [inline services/ui-src/src/measures/2023/shared/CommonQuestions/types.ts:193]: Do we need to delete any of these? Are they being used elsewhere?

**Review by ntsummers1 — APPROVED** [review]
Looks good! Thanks for doing the overview on this. Definitely a lot less impact than my initial impression when I saw the 1000+ line deletion. Other than my nitpick this looks great! Pulled and tested locally.

**Review by keeysnc — DISMISSED** [review]
Ran and tested on my local ~ looks good!

File: services/ui-src/src/measures/2023/shared/CommonQuestions/types.ts (hunk 1 above)
Comment (britt-mo) [reply services/ui-src/src/measures/2023/shared/CommonQuestions/types.ts:193]: good catch

**Review by karla-vm — APPROVED** [review]

**Review by ntsummers1 — APPROVED** [review]

**Pull request merged**